technical-reports/format/index.html

```diff
@@ -74,57 +76,20 @@ <h1>Introduction</h1>
         <em>This section is non normative</em>
       </p>
       <p>
-        Design tokens are a methodology for expressing design decisions in a
-        platform-agnostic way so that they can be shared across different
-        disciplines, tools, and technologies. They help establish a common
-        vocabulary across organisations.
-      </p>
-      <p>
-        There is a growing ecosystem of tools for design system maintainers and
-        consumers that incorporate design token functionality, or would benefit
-        from doing so:
-      </p>
-      <ul>
-        <li>
-          Design tools have begun allowing designers to label and reference
-          shared values for design properties like colors and sizes.
-        </li>
-        <li>
-          Translation tools exist that can convert source design token data into
-          platform-specific source code that can directly be used by developers.
-        </li>
-        <li>
-          Documentation tools can display design token names rather than the raw
-          values in design specs and style guides.
-        </li>
-      </ul>
-      <p>
-        It is often desirable for design system teams to integrate such tools
-        together, so that design token data can flow between design and
-        development tools. For example:
+        Design tokens establish a common vocabulary across organizations. They
```

Comment:
Whoa! Harsh edit :-P

I'm all for making things more concise and using plain English, but I think the specific examples of tools that might use tokens are useful here to set context.